packages/headless/src/test/mock-facet-search-request-options.ts

```diff
@@ -1,12 +1,11 @@
 import {FacetSearchRequestOptions} from '../api/search/facet-search/base/base-facet-search-request';
+import {defaultFacetSearchOptions} from '../features/facets/facet-search-set/facet-search-reducer-helpers';
 
 export function buildMockFacetSearchRequestOptions(
   config: Partial<FacetSearchRequestOptions> = {}
 ): FacetSearchRequestOptions {
   return {
-    captions: {},
-    numberOfValues: 0,
-    query: '',
+    ...defaultFacetSearchOptions,
```

Comment:
The mocks are intentionally set to falsy values to force declaring test values inside the test. This makes the test clearer, and avoids needing to update a bunch of tests that would otherwise break if the default value changes.

Since `initialNumberOfValues` key is not part of the request options, there should be no need to make changes to this file.